bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/cli/CliSpec.java

```diff
@@ -57,31 +53,31 @@ public CliSpec(Option option, Operand[] operands, String... args) {
         // First argument is for the strand
         mainArgs.add(null);
         if (option != null) {
-            BMap<BString, Object> record = option.parseRecord(args);
+            var recordVal = option.parseRecord(args);
             processOperands(option.getOperandArgs());
-            mainArgs.add(record);
+            mainArgs.add(recordVal);
             mainArgs.add(true);
         } else {
-            RecordType type = TypeCreator.createRecordType("dummy", null, 1, new HashMap<>(), null, true, 6);
-            Option dummyOption = new Option(type, ValueCreator.createMapValue(type));
+            var type = TypeCreator.createRecordType("dummy", null, 1, new HashMap<>(), null, true, 6);
+            var dummyOption = new Option(type, ValueCreator.createMapValue(type));
             dummyOption.parseRecord(args);
             processOperands(dummyOption.getOperandArgs());
         }
         return mainArgs.toArray();
     }
 
     private void processOperands(List<String> operandArgs) {
-        int opIndex = 0;
+        var opIndex = 0;
```

Comment:
Why these types are replace by 'var'?

Reply:
It is the new feature in java 10. There's a sonar issue that suggested this change. The type is inferred

Reply:
let's not do this yet, when it comes to style, uniformity matters over improvement.
If we do this, first it needs to be a part of our style guide.

Reply:
It is supported in java 10. I changed only my code. Do you think we shouldn't use this feature in java 10. Between we are using java 11 for builds.

Reply:
I am not against the feature, but I do feel style changing has to happen with community consensus.
If this was a code quality change I would agree that improvement matters over consistency, but for subjective things like style, consistency and community consensus is far more impotent.
You are passionate about coding style, this might be a good opportunity to improve the ballerina code base as a whole, not only your part. wdyt?

Reply:
Will remove the use of vars in the java code as it is not very important, but is only a new feature in java

Reply:
I have changed it back to the old way as requested. But would like to emphasize again that this is not a code style change, but is a new feature in java. I was not aware that community consensus is needed to use a java feature.